src/infra/main/template.yaml

```diff
@@ -1203,6 +1213,8 @@ Resources:
             Location: "../../specs/api.yaml"
       Description: "Private REST API that provides method to interact with the Account Interventions Services"
       StageName: !Ref ApiStageName
+      OpenApiVersion: 3.0.3
+      PropagateTags: true
```

Comment:
Just wondering what the `OpenApiVersion` and `PropagateTags` are for

Reply:
Hey @KG700 
So SAM creates a stage name by default so by having in `OpenApiVersion`  it removes that extra Stage that we have in our PrivateRestApi, but this would only work when deploying the API from scratch, so sadly at the moment having it in, doesn't do much, it doesn't remove the stage as it needs to be added from the beginning, but think good to keep in nonetheless. 

For `PropagateTags` they way I understand it is that when its set to `True` any tags that are applied to the APIGW are also applied to the other underlying resources in the stack.